components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/impl/OAuthOpaqueAuthenticatorImpl.java

```diff
@@ -305,6 +305,9 @@ public OrganizationInfo getOrganizationInfo(String username) throws APIManagemen
             UserStoreManager manager = realm.getUserStoreManager();
             organization =
                     manager.getUserClaimValue(MultitenantUtils.getTenantAwareUsername(username), orgNameClaim, null);
+            if ("super".equals(organization.toLowerCase())) {
+                organization = tenantDomain;
+            }
```

Comment:
Why do you need this?

Reply:
IS 7 sends carbon.super as 'super'. renaming to our convention